pkg/frameworkPackage.targets

```diff
@@ -17,6 +17,7 @@
     <LibFileTargetPath Condition="'$(LibFileTargetPath)' == '' AND '$(PackageTargetRuntime)' != ''">runtimes/$(PackageTargetRuntime)/lib/$(TargetFramework)</LibFileTargetPath>
 
     <NativeFileTargetPath Condition="'$(NativeFileTargetPath)' == '' AND '$(PackageTargetRuntime)' != ''">runtimes/$(PackageTargetRuntime)/native</NativeFileTargetPath>
+    <NativeBinDir Condition="'$(PackageTargetRuntime)' != '' AND $(PackageTargetRuntime.EndsWith('aot'))">$(NativeBinDir)_aot</NativeBinDir>
```

Comment:
I believe that you could simplify this by making your condition be `'$(PackageTargetRuntimeSuffix)' == 'aot'`